manual.md

```diff
@@ -208,12 +207,9 @@ glm::mat4 transform(glm::vec2 const& Orientation, glm::vec3 const& Translate, gl
 
 ### 1.4. Dependencies <a name="section1_4"></a>
 
-When &lt;glm/glm.hpp&gt; is included, GLM provides all the GLSL features it implements in C++.
+The `<glm/glm.hpp>` header provides all standard GLSL features.
 
-There is no dependence with external libraries or external headers such as gl.h, [*glcorearb.h*](http://www.opengl.org/registry/api/GL/glcorearb.h), gl3.h, glu.h or windows.h. However, if &lt;boost/static\_assert.hpp&gt;
-is included, [*Boost static assert*](http://www.boost.org/doc/libs/1_52_0/doc/html/boost_staticassert.html) will be used all over GLM code to provide compiled time errors unless
-GLM is built with a C++ 11 compiler in which case [static\_assert](http://en.cppreference.com/w/cpp/language/static_assert).
-If neither are detected, GLM will rely on its own implementation of static assert.
+GLM does not depend on external libraries or external headers such as `<gl.h>`, [`<glcorearb.h>`](http://www.opengl.org/registry/api/GL/glcorearb.h), `<gl3.h>`, `<glu.h>`, or `<windows.h>`. However, if `<boost/static_assert.hpp>` is included, then [`Boost.StaticAssert`](http://www.boost.org/doc/libs/release/libs/static_assert) will be used to provide compile-time errors.  Otherwise, if using a C++11 compiler, the standard `static_assert` will be used instead. If neither is available, GLM will use its own implementation of `static_assert`.
```

Comment:
If you want to use the proper include path, then `<gl.h>, <glcorearb.h>, <gl3.h> and <glu.h>`
are typically not valid. (unless specific compiler argument to make sur path valid.)
The path should be: `<GL/gl.h>, <GLES3/gl3.h>, <GL/glcorearb.h> and <GL/glu.h>`.